kubernetes/retool-container.yaml

```diff
@@ -63,24 +63,24 @@ spec:
             secretKeyRef:
               name: retoolsecrets
               key: google_client_secret
+        - name: WORKFLOW_TEMPORAL_CLUSTER_NAMESPACE
+          value: "workflows"
+        - name: WORKFLOW_BACKEND_HOST
+          value: http://workflows-api
+        - name: CODE_EXECUTOR_INGRESS_DOMAIN
+          value: http://code-executor:3004
         image: tryretool/backend:X.Y.Z
         name: api
         ports:
         - containerPort: 3000
         resources:
           limits:
-            memory: 4096M
+            memory: 4Gi
+            cpu: 2
           requests:
-            cpu: "1"
-            memory: 2048M
-        volumeMounts:
-        - name: retool-pv
-          mountPath: /retool_backend/pv-data
```

Comment:
Why is the persistent volume removed here?

Reply:
addressed